app/filters.js

```diff
@@ -0,0 +1,46 @@
+const dateFilter = require('nunjucks-date');
+
+module.exports = function (env) {
+  /**
+   * Instantiate object used to store the methods registered as a
+   * 'filter' (of the same name) within nunjucks. You can override
+   * gov.uk core filters by creating filter methods of the same name.
+   * @type {Object}
+   */
+  let filters = {}
+
+  filters.date = dateFilter;
```

Comment:
Why are we exposing this as a filter?

Reply:
The `date` filter is available for the custom `format` param, for example:
```
{%- if item.datetime.type %}
  {{- item.datetime.timestamp | mojDate(item.datetime.type) -}}
{% else %}
  {{- item.datetime.timestamp | date(item.datetime.format) -}}
{% endif -%}
```
If the `type` parameter is provided then `format` is ignored. (Much like the `text` versus `html` params). We favour `type` over `format` and should perhaps be defaulted to "datetime" so provision is not required.